src/components/board/Board.js

```diff
@@ -0,0 +1,45 @@
+import Card from "../card/Card";
+import "./Board.css";
+
+const Board = (props) => {
+  const completedTasks = props.cards.filter((card) => {
+    return card.done === true;
+  });
+  const uncompletedTasks = props.cards.filter((card) => {
+    return card.done === false;
+  });
+  return (
+    <div className="board">
+      <div className="board-header">
+        <h1>Todo List</h1>
+      </div>
+      <div className="board-cards">
+        {uncompletedTasks.map((card) => (
+          <Card
+            key={card.id}
+            id={card.id}
+            title={card.title}
+            isDone={card.done}
+            doneHandler={props.handlers.doneHandler}
+            deleteHandler={props.handlers.deleteHandler}
+          />
+        ))}
```

Comment:
To avoid code redundancy you can concatenate the completed and uncompleted arrays together again after filtering them and put the uncompleted array first 

allTasks = uncompletedTasks.concat(completedTasks)
Then use the allTasks array .map for cards